tests/clients/test_lm.py

```diff
@@ -1,145 +1,24 @@
 from unittest import mock
 
 import pytest
-# from litellm.router import RetryPolicy
 
-# from dspy.clients.lm import LM, _get_litellm_router
+import dspy
+from tests.test_utils.server import litellm_test_server
 
 
-# @pytest.mark.parametrize("keys_in_env_vars", [True, False])
-# def test_lm_chat_respects_max_retries(keys_in_env_vars, monkeypatch):
-#     model_name = "openai/gpt4o"
-#     num_retries = 17
-#     temperature = 0.5
-#     max_tokens = 100
-#     prompt = "Hello, world!"
-#     api_version = "2024-02-01"
-#     api_key = "apikey"
```

Comment:
Let's remove these unused test cases for now and reintroduce them once https://github.com/BerriAI/litellm/issues/6623 is addressed